.changeset/dull-pugs-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@common-fate/terraform-aws-common-fate-deployment": major
+---
+
+remove authz dependency
```

Comment:
```suggestion
For BYOC customers: Removes the Authz service, which has been fully deprecated as of application version 4.3.0, has been removed from the infrastructure.
The supporting DynamoDB table for this service will remain until a future release when it will be removed.
```